Source/WebCore/svg/SVGLengthValue.cpp

```diff
@@ -327,13 +327,19 @@ ExceptionOr<void> SVGLengthValue::setValueAsString(StringView string)
     if (string.isEmpty())
         return Exception { ExceptionCode::SyntaxError };
 
+    // Trim leading and trailing whitespace to match SVG parsing expectations.
+    auto trimmedString = string.toString().trim(isASCIIWhitespace);
+    if (trimmedString.isEmpty())
+        return Exception { ExceptionCode::SyntaxError };
+
```

Comment:
there's a way to do this within the parser machinery.

```c++
auto tokenRange = tokenizer.tokenRange();
range.consumeWhitespace();
range.trimTrailingWhitespace();
```

Reply:
Nice, let me try locally and than can do it this way as well.